waiter/config-full.edn

```diff
@@ -200,8 +200,17 @@
  ;; Waiter supports authenticating requests to identify the principal of the requesting user.
  ;; The principal is then used to validate access to services and also to launch services.
  :authenticator-config {
-                        ;; :kind :one-user allows anonymous access to services (for testing purposes only):
+                        ;; set :kind to one of the configured authenticators
                         :kind :one-user
+
+                        ;; :kind :composite enables using the "authentication" service parameter to
+                        ;; specify the authenticator. Otherwise, the :default-kind authenticator is used .
+                        ;:kind :composite
+                        :composite {:factory-fn waiter.auth.composite/composite-authenticator
+                                    ;; The authenticator to use if one is not specified
+                                    :default-kind :one-user}
```

Comment:
We should explicitly mention the components in here. It feels awkward that I need to manually configure the authenticators I want outside the composite. Also, how do we disallow the default kinds present in settings getting included in components?

Reply:
`default-kind` should be named `default-authenticator` for consistency with how we named it in the composite scheduler.

Reply:
authenticators within composite authenticators will now be called schemes

Reply:
done